frontend/packages/console-dynamic-plugin-sdk/src/extensions/actions.ts

```diff
@@ -1,11 +1,11 @@
 import * as React from 'react';
 import { ExtensionHook, ExtensionK8sKindVersionModel } from '../api/common-types';
 import { ActionContext } from '../api/internal-types';
-import { Extension, ExtensionDeclaration, CodeRef } from '../types';
+import { Extension, CodeRef } from '../types';
 import { AccessReviewResourceAttributes } from './console-types';
 
 /** ActionProvider contributes a hook that returns list of actions for specific context */
-export type ActionProvider = ExtensionDeclaration<
+export type ActionProvider = Extension<
```

Comment:
I've noticed that we currently use `ExtensionDeclaration` for type declaration purpose vs. `Extension` for type guard purpose.

This change aligns both cases to use the standard `Extension` type :+1: